### ISSUE TITLE
[Recoil 0.6.0] fails to install using npm

### ISSUE THREAD

**Issue by saar120** [facebookexperimental/Recoil#1576]
```
npm ERR! code 1
npm ERR! path /Users/saaramrani/Desktop/Fullstack Bootcamp/Workout Tracker/client/node_modules/recoil
npm ERR! command failed
npm ERR! command sh -c cd ./node_modules/gen-flow-files && yarn install && yarn build
npm ERR! sh: line 0: cd: ./node_modules/gen-flow-files: No such file or directory
```

**Comment by gurkerl83** [comment]
Duplicate of https://github.com/facebookexperimental/Recoil/issues/1541

**Comment by JoaoCEAlmeida** [comment]
+1

**Comment by drarmstr** [comment]
Fixed with #1577 and published `0.6.1`
